actionsystemtest/lib/action_system_test/test_helpers/active_job_setup.rb

```diff
@@ -0,0 +1,25 @@
+module ActionSystemTest
+  module TestHelpers
+    # Active Job setup helper for system testing
+    module ActiveJobSetup
+      # Sets the queue adapter to async and remembers the original adapter
+      # setting.
+      #
+      # This method should be used in the test setup code so that jobs run
+      # when the test runs because the test can't wait to check for pass/fail
+      # if it depends on running jobs.
```

Comment:
Yeah, I think that was a legacy setup. We switched to async for both Basecamp and Rails a while back because inline gives you an unrealistic expectation in tests of sequentiality (this is how you hide bugs like scheduling a job within a after_create hook, before the transaction has been committed).

Just confirmed on a new app:

```ruby
$ r c test
Loading test environment (Rails 5.1.0.alpha)
irb(main):001:0> ActiveJob::Base.queue_adapter
=> #<ActiveJob::QueueAdapters::AsyncAdapter:0x007fcda14721d0...
```